src/MCPClient/lib/clientScripts/verifyMD5.sh

```diff
@@ -23,7 +23,7 @@
 # @version svn: $Id$
 
 checkMD5NoGui="`dirname $0`/archivematicaCheckMD5NoGUI.sh"
-date_format="%a %b %e %_H.%M.%S %Z %Y"
+date_args="-u +%Y%m%dT%H%M%SZ"  # 20170605T220452Z (UTC ISO8601)
```

Comment:
The `checkMD5NoGui` is executed once, while the `date` command runs each time that a file is verified, I didn't want to change current behavior.